### PR TITLE
Feature/condo/doma 1500/split sbbol env config

### PR DESCRIPTION
**Pull request by AntonAL**
Depends on #769, which was not merged until infrastructure works will be finished, so, first 16 commits are to skip in review.

Splitted `SBBOL_CONFIG` into `SBBOL_AUTH_CONFIG` and `SBBOL_FINTECH_CONFIG`.
Extracted initialization of `SbbolFintechApi` and removed duplicated code.

### CODE REVIEW

File: apps/condo/domains/organization/integrations/sbbol/SbbolRequestApi.js [hunk 1]
```diff
@@ -25,18 +26,19 @@ const REFRESH_TOKEN_TTL = 30 * 24 * 60 * 60 // its real TTL is 180 days bit we n
  */
 
 class SbbolRequestApi {
-    constructor (accessToken) {
-        // TODO(pahaz): use protected_port?
-        const { protected_host: hostname, port } = SBBOL_CONFIG
```
Comment (pahaz) [inline apps/condo/domains/organization/integrations/sbbol/SbbolRequestApi.js:30]: protected_host can have an own protected_port.

File: apps/condo/domains/organization/integrations/sbbol/oauth2.js [hunk 2]
```diff
@@ -2,29 +2,30 @@
...
 const jwtDecode = require('jwt-decode') // decode jwt without validation
-const dayjs = require('dayjs')
-
+const pino = require('pino')
+const falsey = require('falsey')
+const util = require('util')
 const conf = require('@core/config')
 
-const SBBOL_CONFIG = conf.SBBOL_CONFIG ? JSON.parse(conf.SBBOL_CONFIG) : {}
+const SBBOL_AUTH_CONFIG = conf.SBBOL_AUTH_CONFIG ? JSON.parse(conf.SBBOL_AUTH_CONFIG) : {}
```
Comment (pahaz) [inline apps/condo/domains/organization/integrations/sbbol/oauth2.js:10]: I don't think that's a good idea to change the name of config. You need to copy it on dev and prod. Don't forget it!

File: apps/condo/domains/organization/integrations/sbbol/sync/syncPaymentRequestsForSubscriptions.js [hunk 3]
```diff
@@ -1,54 +1,71 @@
...
 const conf = process.env
-const SBBOL_CONFIG = conf.SBBOL_CONFIG ? JSON.parse(conf.SBBOL_CONFIG) : {}
-const { debugMessage } = require('../common')
+const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
+
+const { logger: baseLogger } = require('../common')
 const { dvSenderFields, BANK_OPERATION_CODE } = require('../constants')
 const { SBBOL_OFFER_ACCEPT_IS_NULL_ERROR, SBBOL_OFFER_ACCEPT_HAS_INCORRECT_PAYER_REQUISITES_ERROR } = require('@condo/domains/subscription/constants/errors')
 
+const logger = baseLogger.child({ module: 'syncPaymentRequestsForSubscriptions' })
```
Comment (pahaz) [inline apps/condo/domains/organization/integrations/sbbol/sync/syncPaymentRequestsForSubscriptions.js:14]: somewhere you use a new logger like for `sbbol/auth` here you use child. Try to be consistent

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/organization/integrations/sbbol/oauth2.js (hunk 2 above)
Comment (AntonAL) [reply apps/condo/domains/organization/integrations/sbbol/oauth2.js:10]: I know, we have discussed, that finally we will have two config variables.

File: apps/condo/domains/organization/integrations/sbbol/sync/syncPaymentRequestsForSubscriptions.js (hunk 3 above)
Comment (AntonAL) [reply apps/condo/domains/organization/integrations/sbbol/sync/syncPaymentRequestsForSubscriptions.js:14]: Changed and made consistent in `26ea5928`

File: apps/condo/domains/organization/integrations/sbbol/SbbolRequestApi.js (hunk 1 above)
Comment (AntonAL) [reply apps/condo/domains/organization/integrations/sbbol/SbbolRequestApi.js:30]: Now it does not have, if it will, I will add it to config.

**Pull request merged**